Cargo.toml

```diff
@@ -13,9 +13,22 @@ Simple and fast implementation of language tag normalization and validation
 """
 edition = "2018"
 
+[features]
+default = []
+serde = ["serde/derive"]
+
+[dependencies.serde]
+version = "1.0"
+optional = true
+
 [dev-dependencies]
 criterion = "0.3"
+serde_json = "1.0"
 
 [[bench]]
 name = "lib"
 harness = false
+
+# enable serde in docs.rs
+[package.metadata.docs.rs]
+features = ["serde"]
```

Comment:
Maybe just `all-features = true` to be safe if we add other features in the future?